src/script/common/c_internal.h

```diff
@@ -34,6 +34,16 @@ extern "C" {
 #include "config.h"
 #include "common/c_types.h"
 
+/*
+	Newer versions of Lua rename lua_objlen to lua_rawlen.
+*/
+#if USE_LUAJIT
+	#if LUAJIT_VERSION_NUMBER < 20100
+		#define lua_rawlen(L, idx) lua_objlen(L, idx)
+	#endif
+#elif LUA_VERSION_NUM < 502
+	#define lua_rawlen(L, idx) lua_objlen(L, idx)
+#endif
```

Comment:
Continuing to use objlen would get us compatibility for free:
<pre>
/usr/include/lua5.3/luaconf.h
383:#define lua_objlen(L,i)		lua_rawlen(L, (i))

/usr/include/lua5.2/luaconf.h
295:#define lua_objlen(L,i)		lua_rawlen(L, (i))
</pre>